server/server.py

```diff
@@ -73,9 +73,19 @@ def check():
 @app.route('/slack/commands', methods=['POST'])
 def handle_commands():
     """Handle rocket slash commands."""
-    txt = request.form['text']
-    uid = request.form['user_id']
-    return core.handle_app_command(txt, uid)
+    logging.info("Slash command received")
+    timestamp = request.headers.get("X-Slack-Request-Timestamp")
+    slack_signature = request.headers.get("X-Slack-Signature")
+    verified = slack_events_adapter.server.verify_signature(
+        timestamp, slack_signature)
+    if verified:

```

Comment:
Please abstract this out into a different function, preferably `verify_slack_message(req)`